rte/kernels-openacc/mo_rte_solver_kernels.F90

```diff
@@ -962,12 +973,12 @@ subroutine adding(ncol, nlay, ngpt, top_at_1, &
     real(wp), dimension(ncol,nlay+1,ngpt), intent(inout) :: flux_dn
     ! ------------------
     integer :: icol, ilev, igpt
-    real(wp), dimension(nlay+1) :: albedo, &  ! reflectivity to diffuse radiation below this level
+    real(wp), dimension(ncol,nlay+1,ngpt) :: albedo, &  ! reflectivity to diffuse radiation below this level
```

Comment:
My understanding from your comments in the Slack is that this is exposing a compiler bug, yes? If so, could you please add comments to that effect, explaining how we'd revert to a lower-memory option if that matters? 

Will you pursue this compiler fix?

Reply:
Right, that part is buggy. About the lower-memory option: the OpenACC compiler should implicitly transform the low-memory variant into what I wrote, because the GPU doesn't have enough per-thread storage. So for the GPU there should be no difference. If you think it is better to use low-memory variable for the CPU, I could easily change that (e.g., use an architecture-dependent macro)

I will file the compiler bug and follow it.

Reply:
Can you add comments to the code documenting this choice? 

The CPU code is distinct, no need to worry about that.

Reply:
Done!